Dockerfile.arm32v6

```diff
@@ -0,0 +1,26 @@
+FROM rust:1.40.0-slim-stretch AS builder
+
+WORKDIR /gatekeeper
+
+COPY ./src            src
+COPY ./.cargo         .cargo
+COPY ./Cargo.lock     .
+COPY ./Cargo.toml     .
+COPY ./rust-toolchain .
+
+# hadolint ignore=DL3015
+RUN apt-get update && \
+    apt-get install -y --install-recommends gcc-arm-linux-gnueabihf=* && \
+    rustup target add arm-unknown-linux-gnueabihf && \
+    cargo build --target=arm-unknown-linux-gnueabihf --release && \
+    cp target/arm-unknown-linux-gnueabihf/release/gatekeeperd /
+
+FROM raspbian/stretch:041518
+
+COPY --from=builder /gatekeeperd /
```

Comment:
COPY --from /gatekeeperd/target/release/gatekeeperd will reduce the image size.

Reply:
@eldesh Does this mean that the image size can be reduced by copying only the binary from the builder?
If it is, it already is.
(Maybe it's a little confusing that I'm copying the binary into the "/" on line 16.)

>     cp target/arm-unknown-linux-gnueabihf/release/gatekeeperd /

Reply:
Sorry. That's exactly what I meant.